test/cli.test.js

```
@@ -2,17 +2,12 @@ require("dotenv").config();
 const { assert } = require("chai");
 const { sleep } = require("../src/utils");
 const mockServer = require("mockttp").getLocal();
+const { writeFileSync, unlinkSync } = require("fs");
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)